### ISSUE TITLE
Purchases by date: SQLSTATE[42000]: Syntax error or access violation

### ISSUE THREAD

**Issue by pixelmachine** [dispositiontools/craft-commerceinsights#10]
Hi @dispositiontools we're getting the following error when running any products by date (`products/purchases`) report.

```
SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #3 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'my_db_name.cvc.title' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
```

This is on Craft 3 and the plugin is up to date. Thanks.

**Comment by dispositiontools** [comment]
Hi @pixelmachine 
I'm very sorry about this - I'll get this sorted this evening and the fix pushed.

**Comment by dispositiontools** [comment]
@pixelmachine why type of database are you using? MySQL or PostgreSQL

**Comment by pixelmachine** [comment]
No, worries! I'm using MySQL

**Comment by dispositiontools** [comment]
Thanks for your patience. I've seen my error and i'm refactoring but it won't be finished until tomorrow now - sorry for the delay.

**Comment by dispositiontools** [comment]
Thank you @pixelmachine this has now been fixed in [v1.0.13](https://github.com/dispositiontools/craft-commerceinsights/releases/tag/v1.0.13) plus a couple of other tiny updates.
